src/dotnet/UI.Blazor.App/Pages/Landing/LandingDocsMenu.razor

```diff
@@ -35,11 +35,16 @@
         Text="Go to Chats"
         Click="@(_ => NavigateTo(Links.Chats))">
     </MenuEntry>
-    <MenuEntry
-        Icon="icon-question-mark-circle"
-        Text="Support"
-        Click="@(_ => NavigateTo("mailto:support@actual.chat"))">
-    </MenuEntry>
+    <a class="ac-menu-item" href="mailto:support@actual.chat" target="_blank">
```

Comment:
doesn't it fall into a pattern with TextContent and IconContent?

Reply:
Link needs tag <a>, not <div>

Reply:
tag a, not div